Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean

```diff
@@ -13,10 +13,11 @@ import Mathlib.RingTheory.MvPowerSeries.LexOrder
 a multivariate power series whose constant coefficient is not a zero divisor
 is itself not a zero divisor
 
-## TODO
+##  Instance
+
+If `R` has `NoZeroDivisors`, then so does `MvPowerSeries σ R`.
```

Comment:
I think this shouldn't be in this file? Or if it does need to be, then it should give a reference to the file which does contain this instance, to avoid confusion

Reply:
No, I think this is the right file. Are you confusing the univariate and multivariate case?

Reply:
Ah, that is what I was doing!